prow/github/app_auth_roundtripper_test.go

```diff
@@ -61,6 +61,67 @@ func (frt *fakeRoundTripper) RoundTrip(r *http.Request) (*http.Response, error)
 func TestAppsAuth(t *testing.T) {
 
 	const appID = "13"
+	testCasesDryRun := []struct {
+		name                string
```

Comment:
Why did you redefine the struct used for the `testCases` and create a new slice rather than just appending your test to the `testCases` slice? Just add a `dryRunClient` parameter to the testCases and use that to configure if the client should be created with dry run mode or not. You can change the client construction from `NewAppsAuthClientWithFields` to `NewClientFromOptions`, that allows to configure the dry run mode